src/photos/lib/onPhotoTrashed.js

```diff
@@ -0,0 +1,52 @@
+import { getReferencedBy } from 'cozy-client'
+import log from 'cozy-logger'
+
+import { buildPhotosTrashed } from 'photos/queries/queries'
+import { DOCTYPE_ALBUMS, DOCTYPE_FILES } from 'drive/lib/doctypes'
+
+const onPhotoTrashed = async client => {
+  log('info', `Service called with COZY_URL: ${process.env.COZY_URL}`)
```

Comment:
to remove :)

Reply:
Is this for data protection reasons? Or to keep the console clean?

Reply:
Both:
- Why do you need to log the cozy_url? 
- We try to avoid to log data within the log. You can do it, you can even log with debug() if needed.

But here, I really don't see the need to log that information.

Reply:
It wasn't just a mimicking of the other service. I was asking about applying a correction to this one too, if it was a matter of privacy. I'm going to correct mine so that I don't need it any more 👍